sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@
 - Removed unused type `AccountResourceType` and `PathLeaseAction`.
 - Changed all previous `LeaseDuration` members to a new type named `LeaseDurationType`.
 - `startsOn` parameter for `GetUserDelegationKey` was changed to optional.
+- Removed `PreviousContinuationToken` from `ListFileSystemsSinglePageResult`.
```

Comment:
How about list files?

Reply:
It does not have that. It is calling DFS endpoint and the REST API does not return that.